core/src/main/java/org/apache/iceberg/PartitionSpecParser.java

```diff
@@ -33,14 +33,19 @@
 import org.apache.iceberg.util.Pair;
 
 public class PartitionSpecParser {
-  private PartitionSpecParser() {
-  }
 
+  private static final String DATA_FILE = "data_file";
+  private static final String NAME = "name";
+  private static final String TYPE = "type";
+  private static final String PARTITION = "partition";
   private static final String SPEC_ID = "spec-id";
   private static final String FIELDS = "fields";
   private static final String SOURCE_ID = "source-id";
   private static final String TRANSFORM = "transform";
-  private static final String NAME = "name";
+  private static final String FIELD_ID = "field-id";
+
+  private PartitionSpecParser() {
+  }
```

Comment:
Please remove any non-functional changes, like this one that moves the private constructor.